actionpack/test/dispatch/request/json_params_parsing_test.rb

```diff
@@ -38,10 +38,13 @@ def teardown
   end
 
   test "does not parse unregistered media types such as application/vnd.api+json" do
-    assert_parses(
-      {},
-      "{\"person\": {\"name\": \"David\"}}", "CONTENT_TYPE" => "application/vnd.api+json"
-    )
+    actual = "{\"person\": {\"name\": \"David\"}}", "CONTENT_TYPE" => "application/vnd.api+json"
```

Comment:
(in case not clear, I've corrected the error, and followed up with a commit that adds comments in possible touch points)